lib/ansible/galaxy/collection/__init__.py

```diff
@@ -1762,6 +1759,18 @@ def _get_file_hash(b_path, filename):  # type: (bytes, str) -> str
         return _consume_file(fp)
 
 
+def remove_or_unlink(path: bytes):
+    """ Removes the symlink/directory/file if it exists. """
```

Comment:
Per PEP 257, it's customary to use imperative mood in docstring title line sentences, for callables:
```suggestion
    """Remove the symlink/directory/file if it exists."""
```